### ISSUE TITLE
FE | Profile | Account Security | Update designs to support 508 updates

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#51085]
### Background

Design tickets: #47299 and #49615

Christina updated the Account Security design to support [accessibility feedback we heard around how headings were handled on the page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/accessibility/accessibility-audit-updates.md#account-security). This ticket is for the frontend implementation of that work.

**State 1**: LOA1 user
- [Desktop mock](https://www.sketch.com/s/ebd4596f-0707-46cb-941e-247a808725cc/a/DPkdo1q)
- [Mobile mock](https://www.sketch.com/s/ebd4596f-0707-46cb-941e-247a808725cc/a/1Kbx0jn)

**State 2**: LOA3 user 
- [Desktop -- account setup partially complete](https://www.sketch.com/s/ebd4596f-0707-46cb-941e-247a808725cc/a/ag4GAJv)
- [Desktop -- account setup fully complete](https://www.sketch.com/s/ebd4596f-0707-46cb-941e-247a808725cc/a/AxAJQkz)
- [Mobile-- account setup partially complete](https://www.sketch.com/s/ebd4596f-0707-46cb-941e-247a808725cc/a/paV8r1G)
- [Mobile-- account setup fully complete](https://www.sketch.com/s/ebd4596f-0707-46cb-941e-247a808725cc/a/VrM0WQA)

### Tasks

(This probably needs grooming)

- [x] Create "Sign in information" block
- [x] Create "Account setup" block and process list
- [x] Make sure LOA1 design shows "Verify your identity" prompt
- [x] Make sure all error states carry over to this design, including fiduciary flag
- [x] Conduct Unit tests
- [x] Run E2E tests covering current code and regression
- [x] Notify PM if this needs to be deployed to Staging behind feature flag

## Acceptance Criteria
- [x] Complete all unit testing
- [x] End-to-end tests showing 0 violations
- [x] All axe checks pass
- [ ] Notify Product Manager that the change is ready to be validated on Staging
- [ ] Confirm PM has opened a new ticket to deploy changes to Production using feature flag, if needed

## Validation
- Can be validated by designer or PM
- [ ] Validate with LOA1 user
- [ ] Validate with LOA3 user
- [ ] Validate with user who has a fiduciary

**Comment by adamwhitlock1** [comment]
Probably wise to put this behind a feature toggle for the redesign. Otherwise seems pretty straightforward.

**Comment by Samara-Strauss** [comment]
> Probably wise to put this behind a feature toggle for the redesign. Otherwise seems pretty straightforward.

Sounds good to me! I'll create a ticket.

**Comment by Samara-Strauss** [comment]
Ticket created for feature flag #51716

**Comment by adamwhitlock1** [comment]
@Samara-Strauss and @BerniXiongA6 I think this is ready for validation.

I turned on the flag for staging and am able to see various states of the new UI:

The 3 ways I have verified it is by using user 0, user 202, and a loa1 user (by creating a new account myself and then logging in)

### `User 0` has ID verified, 2fa present, and has accepted terms and conditions aka is the 'happiest' path:
![Screenshot 2023-01-31 at 12.45.51 PM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/d67d12a3-8299-4073-b4d5-9d56d7214295)

### `User 202` has ID verified, has mhv account, but needs to accept terms and conditions:

![Screenshot 2023-01-31 at 2.33.26 PM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/2f27daff-b452-40ae-bb86-b25bbdd80977)

### An LOA1 `self created user` has nothing shown as complete:

![Screenshot 2023-01-31 at 12.53.50 PM.png](https://images.zenhubusercontent.com/60ddebd504ce2452d0e0fa92/b2516e64-cd7b-4dd0-981a-839e0c910a2b)

**Comment by Samara-Strauss** [comment]
Thanks, @adamwhitlock1! I'm gonna get to this first-thing in the morning.

**Comment by Samara-Strauss** [comment]
This looks good to me! I think we are ready for launch. Is there anything else to do besides flip the feature flag and go to prod, @adamwhitlock1?

## LOA1 Validation

(This is correct since my LOA1 user has 2FA but is obviously not identity verified)

![image](https://user-images.githubusercontent.com/34068740/216087745-2b2322d5-8275-4e83-afd0-a688fd2a71ab.png)

## LOA3 Validation

### User is LOA3 but we can't connect to their records

This is user 0.

![image](https://user-images.githubusercontent.com/34068740/216088533-49771008-0447-4f50-b21d-2492b7c482a3.png)

### User is LOA3, and has completed identity verification and 2FA

This is user 202. I also validated with users 80 & 81, who sees the same view. Neither had completed the T&C, which is fine.

![image](https://user-images.githubusercontent.com/34068740/216089240-fc6a2e04-088a-47b8-957d-2dcb55c1133f.png)

### User is LOA3, but is not a health care user 

This is user 10.

![image](https://user-images.githubusercontent.com/34068740/216089863-882e93ed-26fa-4e8a-ad5e-6a467a839dc4.png)

**Comment by adamwhitlock1** [comment]
Nope, I can test it on prod with my account and then roll it out whenever we are ready.

**Comment by adamwhitlock1** [comment]
Ok we are live on prod!

**Comment by Samara-Strauss** [comment]
YAYYYYY! Validated in prod. Looking great. I'm going to close this. Fantastic work!
